salt/pillar/cmd_yamlex.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 '''
-Execute a command and read the output as YAMLEX. The YAMLEX data is then
-directly overlaid onto the minion's Pillar data
+Execute a command and read the output as YAMLEX. The YAMLEX data is then directly overlaid onto the minion's Pillar data
```

Comment:
I understand other changes but is there a reason to do this line 121 characters long? If there's any I'm ok.

Reply:
There's a note a couple of lines down about this which gave me a clue:
```
# Don't "fix" the above docstring to put it on two lines, as the sphinx
# autosummary pulls only the first line for its description.
```

Perhaps however we don't need that long of a basic description. I can split that long line into two separate lines instead like other docs.